supabase/migrations/20260722095157_sweep_deleted_version_manifests.sql

```
@@ -0,0 +1,145 @@
+-- Sweep soft-deleted app_versions
+
+CREATE INDEX IF NOT EXISTS idx_app_versions_deleted_with_manifest
+  ON public.app_versions (id)
+  WHERE deleted = true AND manifest_count > 0;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)